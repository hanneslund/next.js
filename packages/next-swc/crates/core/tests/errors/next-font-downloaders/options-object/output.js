import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
import '@next/google-fonts?{"font":"ABeeZee"}';
import a from '@next/google-fonts?{"font":"ABeeZee"}';
fn({
    ['variant']: '400'
});
const a = fn({
    10: 'hello'
});
fn({
    variant: [
        ...[],
        'hello'
    ]
});
const a = fn({
    variant: [
        ...[],
        'hello'
    ]
});
fn({
    variant: [
        function() {}
    ]
});
const a = fn({
    variant: [
        i1
    ]
});
fn({
    variant: [
        ,
        ''
    ]
});
const a = fn({
    variant: [
        ,
        ''
    ]
});
fn({
    variant: i1
});
const a = fn({
    variant: ()=>{}
});
fn({
    variant () {}
});
const a = fn({
    variant () {}
});
fn({
    ...{}
});
const a = fn({
    ...{}
});
