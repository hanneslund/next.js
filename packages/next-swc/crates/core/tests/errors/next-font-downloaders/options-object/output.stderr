
  x Unexpected object key type
   ,-[input.js:4:1]
 4 | ABeeZee({ ['variant']: '400' })
   :           ^^^^^^^^^^^
   `----

  x Unexpected object key type
   ,-[input.js:6:1]
 6 | const a = ABeeZee({ 10: 'hello' })
   :                     ^^
   `----

  x Unexpected spread
   ,-[input.js:9:1]
 9 | ABeeZee({ variant: [...[], 'hello'] })
   :                     ^^^
   `----

  x Unexpected spread
    ,-[input.js:11:1]
 11 | const a = ABeeZee({ variant: [...[], 'hello'] })
    :                               ^^^
    `----

  x Unexpected value
    ,-[input.js:14:1]
 14 | ABeeZee({ variant: [function () {}] })
    :                     ^^^^^^^^^^^^^^
    `----

  x Unexpected value
    ,-[input.js:16:1]
 16 | const a = ABeeZee({ variant: [i1] })
    :                               ^^
    `----

  x Unexpected empty value in array
    ,-[input.js:19:1]
 19 | ABeeZee({ variant: [, ''] })
    :                    ^^^^^^
    `----

  x Unexpected empty value in array
    ,-[input.js:21:1]
 21 | const a = ABeeZee({ variant: [, ''] })
    :                              ^^^^^^
    `----

  x Unexpected value
    ,-[input.js:24:1]
 24 | ABeeZee({ variant: i1 })
    :                    ^^
    `----

  x Unexpected value
    ,-[input.js:26:1]
 26 | const a = ABeeZee({ variant: () => {} })
    :                              ^^^^^^^^
    `----

  x Unexpected key
    ,-[input.js:29:1]
 29 | ABeeZee({ variant() {} })
    :           ^^^^^^^^^^^^
    `----

  x Unexpected key
    ,-[input.js:31:1]
 31 | const a = ABeeZee({ variant() {} })
    :                     ^^^^^^^^^^^^
    `----

  x Unexpected spread
    ,-[input.js:34:1]
 34 | ABeeZee({ ...{} })
    :           ^^^
    `----

  x Unexpected spread
    ,-[input.js:36:1]
 36 | const a = ABeeZee({ ...{} })
    :                     ^^^
    `----
